medusa/server/api/v2/config.py

```diff
@@ -279,6 +279,10 @@ def patch(self, *args, **kwargs):
                 theme_name = data['theme']['name']
                 app.THEME_NAME = theme_name
                 done_data['themeName'] = theme_name
+            if key == 'backlogPeriod':
```

Comment:
If you add support for GET, DELETE, PATCH, etc. then you need to add the others. So this needs a `GET` method added.